### ISSUE TITLE
Database setup documentation as-is results in error for zones API

### ISSUE THREAD

**Issue by kevinquinnyo** [PowerDNS/pdns#5382]
Related PR https://github.com/PowerDNS/pdns/pull/5381

 - Program: Authoritative
 - Issue type: Bug report

### Short description
Documentation in markdown/authoritative/howtos.md shows a query that results in an error when attempting to view a zone via the API (I haven't tried anything else besides the API yet).

```
Jun  6 13:52:05 debian pdns_server[1108]: Jun 06 13:52:05 HTTP ISE for "/api/v1/servers/localhost/zones/example.com": STL Exception: Parsing record content (try 'pdnsutil check-zone'): missing field at the end of record content 'localhost admin@example.com 1'
```

### Environment
 - Operating system: Linux debian
 - Software version: 4.0.3-1
 - Software source: Operating system repository ( http://repo.powerdns.com/debian/ jessie-auth-40/main amd64 Packages )

```
$ sudo dpkg -l | grep pdns
ii  pdns-backend-mysql                    4.0.3-1pdns.jessie                   amd64        generic MySQL backend for PowerDNS
ii  pdns-server                           4.0.3-1pdns.jessie                   amd64        extremely powerful and versatile nameserver
```

### Steps to reproduce
1. Follow the 4.x documentation at https://doc.powerdns.com/md/authoritative/howtos/#basic-setup-configuring-database-connectivity particularly adding the "test" records for example.com.
2. Try to view a zone via the api:

```
curl http://localhost:8081/api/v1/servers/localhost/zones/example.com -H 'X-Api-Key: changeme'
```

### Expected behaviour
200 response

### Actual behaviour
500 error

In the daemon log:
```
Jun  6 13:52:05 debian pdns_server[1108]: Jun 06 13:52:05 HTTP ISE for "/api/v1/servers/localhost/zones/example.com": STL Exception: Parsing record content (try 'pdnsutil check-zone'): missing field at the end of record content 'localhost admin@example.com 1'
```

The API response is simply 500 HTTP code and Internal Server Error.

### Other information
This might be related to the following issue:  https://github.com/PowerDNS/pdns/issues/3518

The `pdnsutil check-zone` suggested in the error log actually doesn't complain.

```
$ sudo pdnsutil check-zone example.com.
Jun 06 14:15:59 [bindbackend] Done parsing domains, 0 rejected, 0 new, 0 removed
Checked 7 records of 'example.com', 0 errors, 0 warnings.
```

**Comment by fredleeflang** [comment]
I can confirm this bug. I just upgraded to 4.0.3-1~bpo8+1 from the Debian backports repository and getting the same error on a view zone query. 

/var/log/daemon.log says:

Jul  3 12:56:02 gamma pdns_server[5319]: Jul 03 12:56:02 HTTP ISE for "/api/v1/servers/localhost/zones/3dn.nl": STL Exception: Parsing record content (try 'pdnsutil check-zone'): missing field at the end of record content 'ns1.3dn.nl fredl.3dn.nl 2017070301'

pdnsutil says:

root@gamma:/var/log# pdnsutil check-zone 3dn.nl
Jul 03 12:56:58 [bindbackend] Done parsing domains, 0 rejected, 0 new, 0 removed
Checked 52 records of '3dn.nl', 0 errors, 0 warnings.

**Comment by zeha** [comment]
Please show your exact database contents.

**Comment by fredleeflang** [comment]
Anything in specific or do you want to see the whole database contents?

**Comment by fredleeflang** [comment]
MariaDB [pdns]> select * from records where domain_id=1;
+------+-----------+--------------------------+-------+-----------------------------------------------------------------------+-------+------+-------------+----------+--------------+------+
| id   | domain_id | name                     | type  | content                                                               | ttl   | prio | change_date | disabled | ordername    | auth |
+------+-----------+--------------------------+-------+-----------------------------------------------------------------------+-------+------+-------------+----------+--------------+------+
|    1 |         1 | 3dn.nl                   | SOA   | ns1.3dn.nl fredl.3dn.nl 2017070301                                    | 86400 |    0 |  1331110311 |        0 |              |    1 |
|    2 |         1 | 3dn.nl                   | NS    | ns1.3dn.nl                                                            | 14400 |    0 |  1499072087 |        0 |              |    1 |

**Comment by morozsm** [comment]
I have exactly the same issue when trying to access zone via API or pdnsutil edit-zone:

Error: Parsing record content (try 'pdnsutil check-zone'): missing field at the end of record content 'ns1.msmsoft.com admin.msmsoft.com 2018032001 '

select * from records where id=1\G
*************************** 1. row ***************************
         id: 1
  domain_id: 1
       name: msmsoft.com
       type: SOA
    content: ns1.msmsoft.com admin.msmsoft.com 2018032001 
        ttl: 86400
       prio: 0
change_date: 1357390581
   disabled: 0
       auth: 1
  ordername: NULL
1 row in set (0.00 sec)

pdns-server                          4.1.1-1pdns.xenial

Any ideas guys?

**Comment by breaked01** [comment]
The same issue here... Is there some workaround?

**Comment by Habbie** [comment]
The workaround is to put a full SOA in the database.

**Comment by breaked01** [comment]
> The workaround is to put a full SOA in the database.

Many thanx.

**Comment by glowfishDE** [comment]
> The workaround is to put a full SOA in the database.

Can you describe what you mean by that? When i add a SOA record and fill in all required data, i get this bug. So how can i put a "better/full" SOA record in "the database" ? Do you mean i should add soa records by hand because powerdns-admin is uncapable of creating the correct records in the database ?

**Comment by Habbie** [comment]
> Do you mean i should add soa records by hand because powerdns-admin is uncapable of creating the correct records in the database ?

We don't know what powerdns-admin does - you'll have to check.

**Comment by zeha** [comment]
#5381 fixed the docs; #9593 removed SOA autofilling. I believe this can be closed.
